apps/omg_watcher/lib/omg_watcher/exit_processor.ex

```diff
@@ -232,15 +232,11 @@ defmodule OMG.Watcher.ExitProcessor do
         Utxo.Position.decode(utxo_pos)
       end)
 
-    {:ok, exit_event_triggers, db_updates_from_state, validities} = State.exit_utxos(exits)
+    {:ok, db_updates_from_state, validities} = State.exit_utxos(exits)
+    {new_state, exit_event_triggers, db_updates} = Core.finalize_exits(state, validities)
 
-    _ = if not Enum.empty?(exit_event_triggers), do: Logger.info("Finalized exits: #{inspect(validities)}")
+    EventerAPI.emit_events(exit_event_triggers)
```

Comment:
Let's leave the logging. We can decrease logging level to debug.

Reply:
https://github.com/omisego/elixir-omg/pull/569/files#diff-f1ce1871bf4b96d60bda0fbd02e3891cR226 there's logging, there were two invocations of logging, I only removed one :)